SmartTutor/SmartTutor.csproj

```diff
@@ -14,6 +14,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
Why do we need this dependency? System.Text.Json should be quite enough, and I'm not a fan of introducing libraries when the base framework can provide all the functionality.

In general we should minimize additional code brought into our platform as that can cause issues with licensing. This is not the case for this particular package, but once again the base framework support should be enough.